apps/jobs/models.py

```diff
@@ -50,7 +50,8 @@ class Submission(TimeStampedModel):
     input_file = models.FileField(upload_to=input_file_name)
     stdout_file = models.FileField(upload_to=stdout_file_name, null=True, blank=True)
     stderr_file = models.FileField(upload_to=stderr_file_name, null=True, blank=True)
-
+    execution_time_limit = models.PositiveIntegerField(default=timedelta(seconds=300))
```

Comment:
Please change it to `execution_time_limit = models.PositiveIntegerField(default=300)`